test/mithril.route.js

```diff
@@ -130,68 +129,7 @@ describe("m.route()", function () {
 		expect(spy).to.not.have.been.called
 	})
 
-	// FIXME: this causes others to fail
-	xdit("skips route change if subcomponent ctrl.onunload calls preventDefault", function (root) { // eslint-disable-line
-		mode("search")
-
-		var spy = sinon.spy()
-
-		var subsub = {
```

Comment:
Actually, the other two are missing this layer of indirection. Please restore these tests and re-enable them if they're passing now (i.e. not causing problems).